include/FLAC/format.h

```diff
@@ -282,14 +282,24 @@ extern FLAC_API const char * const FLAC__SubframeTypeString[];
 /** CONSTANT subframe.  (c.f. <A HREF="../format.html#subframe_constant">format specification</A>)
  */
 typedef struct {
-	FLAC__int32 value; /**< The constant signal value. */
+	FLAC__int64 value; /**< The constant signal value. */
 } FLAC__Subframe_Constant;
 
+/** An enumeration of the possible verbatim subframe data types. */
+typedef enum {
+	FLAC__VERBATIM_SUBFRAME_DATA_TYPE_INT32, /**< verbatim subframe has 32-bit int */
+	FLAC__VERBATIM_SUBFRAME_DATA_TYPE_INT64 /**< verbatim subframe has 64-bit int */
+} FLAC__VerbatimSubframeDataType;
+
 
 /** VERBATIM subframe.  (c.f. <A HREF="../format.html#subframe_verbatim">format specification</A>)
  */
 typedef struct {
-	const FLAC__int32 *data; /**< A pointer to verbatim signal. */
+	union {
+		const FLAC__int32 *int32; /**< A FLAC__int32 pointer to verbatim signal. */
```

Comment:
Bad choice of member name: many C sources out there define `int32` and/or `int64` types for themselves.

Reply:
`data32` and `data64` maybe???

Reply:
Thanks for the feedback. Do you think the general idea (having a union in a struct with a member indicating which union member is being used) uses the right concept for the job here?

Reply:
A simple `void*` pointer cast to correct type where it is used would have done the job.

Reply:
Thanks for the suggestion. I'll try and see whether that makes the code cleaner or not.